### DISCUSSION TITLE
Disable multi-selection in table

### DISCUSSION THREAD

**Discussion by kmurari103** [iTwin/iTwinUI-react#486] [Q&A] [answered]
Hi there,
I'm using [this storybook example](https://itwin.github.io/iTwinUI-react/?path=/story/core-table--selectable) to make rows selectable but i noticed it support multiple selection. I need to keep selection restricted to one row at a time. Is there additional props that needs to be passed alongwith `isSelectable={true}` in `<Table>` component?

**Comment by bentleyvk** [comment, marked as answer]
Hi,
It seems related to this issue: https://github.com/iTwin/iTwinUI-react/issues/155

**Comment by kmurari103** [comment]
Thanks @bentleyvk , that worked well.
Just one more thing, is it possible to highlight the row when a marker/icon is clicked in the viewport? I mean now when i select a row- it gets highlighted. I also want to highlight the respective row when an icon/marker is selected in viewport.

**Reply by mayank99** [reply]
You can use manual selection state for that. https://itwin.github.io/iTwinUI-react/?path=/story/core-table--controlled-state
